README.md

```diff
@@ -27,7 +27,7 @@ FairScale supports:
 
 ## Requirements
 
-* PyTorch >= 1.5.1
+* PyTorch >= 1.6.0
```

Comment:
I remember setup.py and project.toml have some torch version numbers too. Also, requirements*.txt too?